Firestore/tests/System/FirestoreMultipleDbTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Firestore\Tests\System;
+
+use Google\Cloud\Firestore\FieldPath;
+use Google\Cloud\Firestore\Query;
+
+/**
+ * @group firestore
+ * @group firestore-multipledb
+ */
+class FirestoreMultipleDbTest extends FirestoreTestCase
+{
+    private $document;
+
+    public function setUp(): void
+    {
+        $this->document = self::$multiDbCollection->newDocument();
+    }
+
+    public function testInsert()
+    {
+        $this->assertFalse($this->document->snapshot()->exists());
+
+        self::$multiDbClient->runTransaction(function ($t) {
+            $t->create($this->document, [
+                'foo' => 'bar'
+            ]);
+        });
+
+        $this->assertTrue($this->document->snapshot()->exists());
+    }
+
+    public function testUpdate()
+    {
+        $this->document->create([
+            'foo' => 'bar'
+        ]);
+
+        self::$multiDbClient->runTransaction(function ($t) {
+            $t->update($this->document, [
+                ['path' => 'bat', 'value' => 'baz']
+            ]);
+        });
+
+        $this->assertEquals([
+            'foo' => 'bar',
+            'bat' => 'baz'
+        ], $this->document->snapshot()->data());
+    }
+
+
+    public function testCollectionGroup()
+    {
+        list ($query) = $this->createDocuments([
```

Comment:
Why don't we just return `$query` here? I see we return other information like `$paths` and `$collectionGroup`, but... why? This is the only place this method is called, so  we can keep it simple and just return `$query`.

... also as an FYI, `list` can now be replaced with array syntax (which is now more standard):
```suggestion
        [$query] = $this->createDocuments([
```

Reply:
Originally, I planned to add more cases, but I realized this maybe sufficient. I will fix this.